docs/trulens/contributing/techdebt.md

```diff
@@ -12,8 +12,9 @@ lives.
 
 See `instruments.py` docstring for discussion why these are done.
 
-- We inspect the call stack in process of tracking method invocation. It may be
-  possible to replace this with `contextvars`.
+- __Stack walking removed in favor of contextvars in 1.0.2__. We inspect the
```

Comment:
nit: released 1.0.2 yesterday, bump to either 1.0.3 or 1.1?

Reply:
fixed